theseus/geometry/functorch.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import threading
+from typing import Any
+
+
+class _FunctorchContext(object):
```

Comment:
I don't think we are following the convention to inherit from `object`.

Reply:
This is only needed for compatibility with Python 2, AFAIK, so we don't need this.

Reply:
Removed.